### ISSUE TITLE
pexpect.exceptions.TIMEOUT on running seautomate

### ISSUE THREAD

**Issue by chris-h1** [trustedsec/social-engineer-toolkit#796]
I installed setoolkit on new debian 10 from github and wanted to run seautomate.

**Expected Behaviour**

seautomate should start and run an answer file

**Actual Behaviour**

[!] The python-pycrypto python module not installed. You will lose the ability for encrypted communications.

[*] Spawning SET in a threaded process...
[*] Something went wrong, printing error: Timeout exceeded.
<pexpect.pty_spawn.spawn object at 0x7fe013c0b110>
command: /usr/bin/python3
args: ['/usr/bin/python3', 'setoolkit']
buffer (last 100 chars): 'y use this tool for lawful purposes only.\x1b[0m\r\n\x1b[92m\r\n\r\nDo you agree to the terms of service [y/n]: '
before (last 100 chars): 'y use this tool for lawful purposes only.\x1b[0m\r\n\x1b[92m\r\n\r\nDo you agree to the terms of service [y/n]: '
after: <class 'pexpect.exceptions.TIMEOUT'>
match: None
match_index: None
exitstatus: None
flag_eof: False
pid: 7864
child_fd: 6
closed: False
timeout: 30
delimiter: <class 'pexpect.exceptions.EOF'>
logfile: None
logfile_read: None
logfile_send: None
maxread: 2000
ignorecase: False
searchwindowsize: None
delaybeforesend: 0.05
delayafterclose: 0.1
delayafterterminate: 0.1
searcher: searcher_re:
    0: re.compile('99\\) Exit the Social-Engineer Toolkit')

**Set Version**

Version: 8.0.3

Where should the <y> be inserted for accepting the lawful purposes?

**Comment by chris-h1** [comment]
**Temporary solution**

/usr/bin/python3 /usr/share/setoolkit/seautomate <answer-file>

**Comment by HackingDave** [comment]
Addressed in SET 8.1.1 as part of the packaging, dependency, and Python runtime modernization. Please pull latest master and open a new issue with current reproduction steps if the problem still occurs.
